Shapes/index.js

```diff
@@ -0,0 +1,35 @@
+
+/*   Area Of Circle   */
+
+const getAreaOfCircle = (radius) => {
+    return Math.PI * radius ** 2;
+}
+
+let circleArea = getAreaOfCircle(25)
+console.log(circleArea)
+
+/*      Circumference Of Circle        */
+
+const getCircumferenceOfCircle = (radius) => {
+    return 2 * Math.PI * radius
+}
+let circumfernce = getCircumferenceOfCircle(25)
+console.log(circumfernce)
```

Comment:
Another way to organize code is to define functions at the top of the file, and then call them at the bottom

```suggestion
const getAreaOfCircle = (radius) => {
    return Math.PI * radius ** 2;
}

const getCircumferenceOfCircle = (radius) => {
    return 2 * Math.PI * radius
}

let circleArea = getAreaOfCircle(25)
console.log(circleArea)

let circumfernce = getCircumferenceOfCircle(25)
console.log(circumfernce)
```